solidity/contracts/isms/hook/layer-zero/LayerZeroV2Ism.sol

```diff
@@ -79,7 +79,18 @@ contract LayerZeroV2Ism is AbstractMessageIdAuthorizedIsm {
     function _messageId(
         bytes calldata _message
     ) internal pure returns (bytes32) {
-        return bytes32(_message[FUNC_SELECTOR_OFFSET:]);
+        return bytes32(_message[FUNC_SELECTOR_OFFSET:ORIGIN_SENDER_OFFSET]);
```

Comment:
fwiw I dont think this change is actually necessary because the compiler will truncate the bytes32 but I like the explicitness

Reply:
any reason why we are not using a library for this?